### PR TITLE
fix: adding test cases

### PR DESCRIPTION
**Pull request by rachanamamillapalli**
PR checklist:

- [ ] Github Issue associated with branch.
- [ ] Code passes linting through autopep8.
- [ ] PR branch is up-to-date with latest master branch.

Step 1 for #5

### CODE REVIEW

File: tests/algorithms/lstm_classifier/lstm_classifier_test.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                "Label-Train-1",
+                "Label-Train-2",
+                "Label-Train-3",
+                "Label-Train-4",
+                "Label-Validation-5",
+            ],
+        )
+
+        # Check if the first 4 articles are a part of training data (0.8)
+        self.assertEqual(len(result[0]), 4)
```
Comment (MadaraUchiha-314) [inline tests/algorithms/lstm_classifier/lstm_classifier_test.py:40]: The numbers `4` and `1` should not be hard-coded. There is a variable exposed in the file [lstm_classifier.py#L36] called `TRAINING_PORTION`. Use that. The point being, the tests should not fail if i go and change the ratio.

[lstm_classifier.py#L36]: https://github.com/intuit/fawkes/blob/master/src/algorithms/lstm_classifier/lstm_classifier.py#L36

File: tests/algorithms/lstm_classifier/lstm_classifier_test.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        review = [
+            {
+                "message": "I just heard about this budgeting app and I gave it a try and done.",
+                "derived-insight": {"category": "Application"},
+            }
+        ]
+        result = lstm_classifier.get_articles_and_labels(review)
+
+        # Check if stop words have been removed from the review message
+        self.assertFalse("and" in result[0][0])
```
Comment (MadaraUchiha-314) [inline tests/algorithms/lstm_classifier/lstm_classifier_test.py:22]: Can we also check that it hasn't removed the non stop-words ?

File: tests/algorithms/lstm_classifier/lstm_classifier_test.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+        # Check if review category is the same as label returned
+        self.assertEqual(result[1][0], "Application")
```
Comment (Konoha-14) [inline tests/algorithms/lstm_classifier/lstm_classifier_test.py:25]: This test doesn't check the case when `category` has non-word characters (non-alphanumeric). 
you can test with: 
```javascript 
{ 
 "message": "Don't make me sign in everytime.",
 "derived-insight": {"category": "sign-in/sign-up"},
}
```

File: tests/algorithms/lstm_classifier/lstm_classifier_test.py (hunk 2 above)
Comment (Konoha-14) [reply tests/algorithms/lstm_classifier/lstm_classifier_test.py:22]: Can we also check if NA strings have been removed?

File: tests/algorithms/lstm_classifier/lstm_classifier_test.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+import os
+from unittest.mock import MagicMock
+
+sys.path.append(os.path.realpath("."))
+
+import src.algorithms.lstm_classifier.lstm_classifier as lstm_classifier
+
+
+class LstmClassifierTest(unittest.TestCase):
+    def test_get_articles_and_labels(self):
```
Comment (Konoha-14) [inline tests/algorithms/lstm_classifier/lstm_classifier_test.py:12]: Can you also check `original_label_to_clean_label` which is returned.

[returned]: https://github.com/intuit/fawkes/blob/8f4a0183b20efebc973ad6157da599439ad9dcc2/src/algorithms/lstm_classifier/lstm_classifier.py#L78

File: tests/algorithms/lstm_classifier/lstm_classifier_test.py (hunk 2 above)
Comment (rachanamamillapalli) [reply tests/algorithms/lstm_classifier/lstm_classifier_test.py:22]: Are we removing or just ignoring the NA strings? [here](https://github.com/intuit/fawkes/blob/8f4a0183b20efebc973ad6157da599439ad9dcc2/src/algorithms/lstm_classifier/lstm_classifier.py#L69)

Comment (rachanamamillapalli) [reply tests/algorithms/lstm_classifier/lstm_classifier_test.py]: @Konoha-14 article.strip() doesn't seem to be enough? When the test message is " " it fails to clean vs when the message is just "" (Note the space missing).

**Review by Konoha-14 — APPROVED** [review]

**Pull request merged**